NebulousConquestHelper/src/BackingXmlFile.cs

```diff
@@ -41,6 +41,14 @@ public T Object
 					XmlSerializer serializer = new XmlSerializer(typeof(T));
 					T loaded = (T)serializer.Deserialize(stream);
 					stream.Close();
+					if (loaded is Backed<T> backed)
+					{
+						backed.BackingFile = this;
+						if (backed is T again)
```

Comment:
Why do we need to check if `backed` is T, when we just casted it from `loaded` which is T? And then store it in `again` when we could just return `loaded`?

Reply:
Hm. You're right, I don't need to do this because casting just changes the type, not the object. So setting the BackingFile mutates the original object. I'll update.

Reply:
Updated.